apps/state/config/config.exs

```diff
@@ -223,6 +223,8 @@ config :state, :stops_on_route,
     "Shuttle-NewtonHighlandsWellesleyFarms-0-" => true,
     # Needham Line shuttle
     "Shuttle-ForestHillsNewtonHighlands-0-" => true,
+    #Kingston-Halifax shuttle
```

Comment:
I think CI is complaining about there not being a space after your leading #... 😅